app/views/users/_notifications.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= form_for @user do |f| %>
+<%= form_for @user.notification_setting do |f| %>
```

Comment:
I wanted to verify a suspicion I have about this, but I can't see where this partial is rendered. Do you know where I have to go to see this form rendered?

My suspicion is that this won't match any known route since the `notification_settings` route is namespaced under `users`.

Reply:
Okay, I just found a way to verify that this does indeed work in the Rails console:

```
[7] pry(main)> Rails.application.routes.url_helpers.url_for User.last.notification_setting
=> "http://localhost:3000/users/notification_settings/13564"
```

It looks like it's because the model class is `Users::NotificationSetting`, so it gets URL-path-namespaced under `/users`, too, which I did not realize was a thing Rails did.

The confusion about what renders the partial is still there, though. 😄 Is that part of what you'll be migrating to in another phase of this project?

Reply:
@jgaskins this PR merges back into https://github.com/forem/forem/pull/13904 - that's probably why you can't find the code :)

Reply:
@rhymes Yep. That explains it! :joy: Thanks!